frontend/src/pages/Listing.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import ListingGrid from 'src/components/Listing/ListingGrid';
+import PageHeader from 'src/components/PageHeader';
+
+export interface IListingProps {
+}
+
+export default function Listing(props: IListingProps) {
+    const listing = [
```

Comment:
Consider renaming this to AllListings or ListingsBrowser as Listing implies a single Listing that might get confused with ListingCard

Reply:
changed this naming